stdlib/blocktx.sol

```diff
@@ -0,0 +1,69 @@
+// SPDX-License-Identifier: Apache-2.0
+
+function blockhash(uint blockNumber) returns (bytes32 ret) {
+  assembly {
+    ret := blockhash(blockNumber)
+  }
+}
```

Comment:
Just a heads up before we drop the source in the repo - I think that the piece of code as prominent as the standard library should really follow our own coding style :)

```suggestion
function blockhash(uint _blockNumber) returns (bytes32 ret) {
    assembly {
        ret := blockhash(_blockNumber)
    }
}
```

Reply:
Makes sense. Do we have any good guide for variable names (i.e. for `ret`)?